src/wallet/create_wallet_flow.h

```diff
@@ -32,13 +31,10 @@
 
 /**
  * @brief This API executes the wallet creation flow on the device
- * @details This function generate random mnemonics or inputs seed phrase from
- * user, takes user inputs for wallet configuration, writes the wallet shares on
- * the X1 vault flash and X1 cards and verifies each share
- *
- * @param new_wallet true if a new wallet needs to be created, false if the user
- * wants to recover the wallet from a seed phrase
+ * @details This function generate random mnemonics, takes user inputs for
+ * wallet configuration, writes the wallet shares on the X1 vault flash and X1
+ * cards and verifies each share
  */
-void create_wallet_flow(bool new_wallet);
+void create_wallet_flow(void);
```

Comment:
Some renaming could be done here, file and function both can be misleading now as the purpose has changed.

Reply:
Makes sense, updated

Reply:
Updated the name